index.bs

```diff
@@ -3501,15 +3501,17 @@ closed</dfn> steps given |window|, |accepted| and optional |user text|
         browsingContext.UserPromptOpenedParameters = {
           context: browsingContext.BrowsingContext,
           type: "alert" / "confirm" / "prompt" / "beforeunload",
-          message: text
+          message: text,
+          ? defaultValue: text
         }
       </pre>
    </dd>
 </dl>
 
 <div algorithm>
 The [=remote end event trigger=] is the <dfn export>WebDriver BiDi user prompt
-opened</dfn> steps given |window|, |type| and |message|.
+opened</dfn> steps given |window|, |type|, |message|, and optional |default value|
+(default: null).
```

Comment:
Note that this addition will also [require a HTML spec change](https://html.spec.whatwg.org/#simple-dialogs:webdriver-bidi-user-prompt-opened).

Do we need `(default: null)` here?

Reply:
The HTML spec PR is linked to this issue (direct link: https://github.com/whatwg/html/pull/9645)

Reply:
Is there a place I missed with the HTML spec PR I attached in this PR? https://github.com/whatwg/html/pull/9645

I looked at `WebDriver BiDi user prompt closed` where is the other optional parameter `userText`.
There we explicitly say that the default is null.

The other option is to patch the HTML the way it was done for `message` -> always pass empty string, but considering that we have this only for one of the events I think it's better if  we omit it.